dianna/visualization/image.py

```diff
@@ -62,9 +68,11 @@ def plot_image(heatmap,
                    labelbottom=False,
                    labelright=False,
                    labeltop=False)
-    if show_plot:
-        plt.show()
+
```

Comment:
If I understand correctly the logic you'd like to always show the plot by default (that's why `show_plot` is `True` by default). Then in the code, it's a bit strange that the only use of the parameter is when it is **not** true. Isn't therefore better for readability `hide_plot` (with default `False`) a better name? Then the `if` statement will read more logically. Just a suggestion.

Reply:
@elboyran I agree with you. When I was refactoring this part, I was also thinking in the same way. I noticed that this interface of visualization functions i.e. `show_plot` as an input argument has already been used in many places e.g. code, tests, notebooks, .... Renaming it here adds a lot of changes to this pull request that was not initially intended. If you agree, I skip renaming it in here, but I submit an issue for it.

Reply:
I understand your reasoning. I'm not sure it's actually better (maybe only slightly) of the presence of another parameter, output_filename, also there to output something. Also hide_plot sounds a bit like a negative, which would make hide_plot=False make a double negative. I'm sort of fine with either option but I would suggest not spend time on this and just leave it as it is.

Reply:
OK.